src/hooks/use_send_message.rs

```diff
@@ -85,32 +87,35 @@ pub fn use_send_message(cx: &ScopeState) -> &UseSendMessageState {
             messages,
             session,
             message_dispatch_id,
-            message_factory
+            message_factory,
+            public
         ];
 
         async move {
             while let Some(message_item) = rx.next().await {
                 if message_item.msg.starts_with('!') {
-                    let Err(error) = handle_command(&message_item, &client.get()).await else {
-                        return;
-                    };
+                    match handle_command::handle_command(&message_item, &client.get()).await {
+                        Ok(Command::Join(_)) => {}
```

Comment:
Am I understanding correctly, you create the logic of the join command but don't use it?

Reply:
The command can be used, only there is no feedback for the user for now, it's a command that has been there since the origins of the app but for now I do not think it's such a priority to complete. Maybe could create a issue to remember that it's here.